### DISCUSSION TITLE
More app icon options.

### DISCUSSION THREAD

**Discussion by zerodrek** [javikr/sofatime#52] [Ideas]
I know I’m weird but I’d like to see more app icons that are similar to the Black & White icon 
![IMG_5625](https://github.com/javikr/sofatime/assets/2582474/d5205e0c-4d50-41c4-ab61-ea3576fa3dda) with the white icon in the middle but with different colored backgrounds to make the app looks more like it could be a stock app and fit in more with other apps and darker phone wallpapers.

**Comment by javikr** [comment]
Hi there,

Great idea. We are working on more app icons
